### PR TITLE
fix: Remove minted out collection from landing

### LINKED ISSUE #10664: Remove minted out collection from landing
Here, you can see on landing that Tilescapes are minted out. I would filter it in a way that if the collection is mint out, it will not be shown on the landing anymore - instead of that other collection from the list can take its place.

Screenshot 2024-07-22 at 12 20 09

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #10664

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/drops/useDrops.ts [hunk 1]
```diff
@@ -48,28 +49,33 @@ const DROP_LIST_ORDER = [
...
     const formattedDrops = await Promise.all(
-      dropsList.value.map(async (drop) => getFormattedDropItem(drop, drop)),
+      dropsList.value.map(async drop => getFormattedDropItem(drop, drop)),
     )
 
     drops.value = formattedDrops
 
+    if (filterOutMinted) {
+      drops.value = drops.value.filter(drop => !drop.isMintedOut)
+    }
```
Comment (hassnian) [inline components/drops/useDrops.ts:70]: nitpick , this avoids updating the ref twice

``````
  let formattedDrops = await Promise.all(
      dropsList.value.map(async drop => getFormattedDropItem(drop, drop)),
    )

    if (filterOutMinted) {
      formattedDrops = formattedDrops.filter(drop => !drop.isMintedOut)
    }

    drops.value = formattedDrops
``````

or maybe use a `then` and this way you don't have to use `let` and override

**Review by hassnian — COMMENTED** [review]
would be nice if we could filter this out on the worker side 

then we wouldn't have to keep track of minted out drops and update the limit here

**Comment by Jarsen136** [comment]
> would be nice if we could filter this out on the worker side
> 
> then we wouldn't have to keep track of minted out drops and update the limit here
> 
> 

Yes, I have already thought about it. The reason stop me from implementing it is that the worker side does not have the correct stats of `minted`.

File: components/drops/useDrops.ts (hunk 1 above)
Comment (Jarsen136) [reply components/drops/useDrops.ts]: updated

**Review by hassnian — APPROVED** [review]

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
You have conflicts @Jarsen136

**Pull request merged**